src/libs/actions/PersonalDetails.js

```diff
@@ -57,6 +57,12 @@ function getAvatar(personalDetail, login) {
 function getDisplayName(login, personalDetail) {
     const userDetails = personalDetail || personalDetails[login];
 
+    // If we have a number like +15857527441@expensify.sms then let's remove @expensify.sms
+    // so that the option looks cleaner in our UI.
+    if (login.includes(CONST.LOGIN.EXPENSIFY_SMS_SUFFIX)) {
```

Comment:
This is probably all you need -> https://github.com/Expensify/expensify-common/blob/d1bfacca6fb92f7e89a2a9cdf0863fad6481c0f2/lib/str.js#L909-L918

Reply:
Oh i missed that Str library probably has checks for these. I think I'll need both Str isSMSLogin and removeSMSDomain so shall update.